LICENSE-MIT

```diff
@@ -1,22 +0,0 @@
-Copyright (c) 2012 Djordje Lukic, http://rumpl.org
```

Comment:
question: why remove the license?

Reply:
I went on a bit of a late night directory cleaning spree, and got carried away. I'll recommit it.